requirements.txt

```diff
@@ -42,3 +42,4 @@ kafka-python
 importlib-metadata==3.7.0; python_version < "3.8"
 tenacity==6.3.1
 tzlocal<3.0
+scp
```

Comment:
Is there any version requirement for `scp` ?

Reply:
done